paparazzi-gradle-plugin/src/test/java/app/cash/paparazzi/gradle/PaparazziPluginTest.kt

```diff
@@ -1387,6 +1387,18 @@ class PaparazziPluginTest {
     )
   }
 
+  @Test
+  fun jacoco() {
+    val fixtureRoot = File("src/test/projects/jacoco")
+
+    gradleRunner
+      .withArguments("testDebug", "--stacktrace")
+      .runFixture(fixtureRoot) { build() }
```

Comment:
I guess there's no way to ensure that this is running on Java 17. It's just the way things are because of CI/AGP.